cmd/airplane/main.go

```
@@ -42,6 +41,12 @@ func main() {
 		}
 		logger.Log("")
 
+		// TODO: is this too noisy?
+		sentryID := sentry.CaptureException(err)
```

Doesn't sentry allow us to filter/ignore errors from their UI? if so, we can remove the TODO?

---

Yeah that's true! Let's try that first. In my experience the "ignore" parts of Sentry have been more as a last resort, since you still have to ignore them as they come up and they still count towards your quota. But we can give it a shot first.